common/flatpak-transaction-log.h

```diff
@@ -0,0 +1,51 @@
+/* flatpak-transaction-log.h
+ *
+ * Copyright © 2017 Endless Mobile, Inc
+ *
+ * This file is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authors:
+ *       Sam Spilsbury <sam@endlessm.com>
+ */
+
+#ifndef FLATPAK_TRANSACTION_LOG_H
+#define FLATPAK_TRANSACTION_LOG_H
+
+#include <string.h>
+
+#include <gio/gio.h>
+#include <glib-object.h>
+
+#define FLATPAK_TYPE_TRANSACTION_LOG flatpak_transaction_log_get_type ()
```

Comment:
Nitpick: Conventionally, this should be within the `G_BEGIN_DECLS` section. Stick it on the line above the `G_DECLARE_FINAL_TYPE`.

Reply:
Yeah, I wasn't exactly certain where it went in relation to `G_DECLARE_FINAL_TYPE`. I'll move it.